docs/lifecycle/lfn_entry/opx.md

```diff
@@ -1,2 +1,285 @@
-== OpenSwitch(OPX) Project Data ==
-**website** www.openswitch.net
+# OpenSwitch (OPX) Project Data
+
+## Instructions
+
+This document is a template to collect the data LFN thinks are relevant to LFN Project health. Projects will copy this template and instantiate it with their data as a part applying for lifecycle state transitions.
+
+`mkdir -p <relevant lifecycle transition dir>/<project name>/ cp project_data_template.rst !$/project_data.rst`
+
+If the project has instantiated this template before they are welcome to start from that base and update it with any new information. Be sure that the data requested by the primary copy of the Project Data Template has not changed.
+
+These instructions should be removed from the instantiated template.
+
+Here is the file for Tungsten Fabric. It’s a work in progress too: 
+
+[https://gerrit.linuxfoundation.org/infra/gitweb?p=lfn%2Fprocess.git;hb=refs%2Fchanges%2F80%2F12880%2F1;f=docs%2Flifecycle%2Flfn_entry%2Ftungsten_fabric%2Fproject_data.rst](https://gerrit.linuxfoundation.org/infra/gitweb?p=lfn%2Fprocess.git;hb=refs%2Fchanges%2F80%2F12880%2F1;f=docs%2Flifecycle%2Flfn_entry%2Ftungsten_fabric%2Fproject_data.rst)  
```

Comment:
use 

[Tungsten Fabric project data](https://gerrit.linuxfoundation.org/infra/gitweb?p=lfn%2Fprocess.git;hb=refs%2Fchanges%2F80%2F12880%2F1;f=docs%2Flifecycle%2Flfn_entry%2Ftungsten_fabric%2Fproject_data.rst) 

or  just 
https://gerrit.linuxfoundation.org/infra/gitweb?p=lfn%2Fprocess.git;hb=refs%2Fchanges%2F80%2F12880%2F1;f=docs%2Flifecycle%2Flfn_entry%2Ftungsten_fabric%2Fproject_data.rst

The same to others.